passion.css

```diff
@@ -0,0 +1,53 @@
+body {
+    margin: 0;
+}
+
+.header {
+    display: block;
+    width: 850px;
```

Comment:
what is the problem to add `width` property to the body and do not `ctrl + c` + `ctrl + v` this line in other 5+ blocks ?